modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase

```diff
@@ -201,12 +201,44 @@ definition {
 	}
 
 	@description = "LPS-135393 - Verify the table is updated when changing a page on the Custom Object Portlet"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test TableUpdatedWhenChangingPage {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Text",
+			fieldName = "customObjectField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		for (var number : list "1,2,3,4,5") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "text",
+				objectName = "CustomObject",
+				value = "Test text");
+		}
+
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject");
+
+		ObjectPortlet.changePagination(itemsPerPage = "4 items");
+
+		ObjectPortlet.checkAnyPage(pageNumber = "2");
+
+		ObjectPortlet.assertEntryRowIsPresent(rowNumber = "1");
+
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "2");
+
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "3");
 
-		// TODO LPS-138961 TableUpdatedWhenChangingPage pending implementation
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "4");
 
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "5");
```

Comment:
Use `ObjectPortlet.viewEntry` and `ObjectPortlet.assertEntryNotPresent` here.

Check this as reference: https://github.com/rodrigocunhaa/liferay-portal/pull/1038/files